Bracket.java

```diff
@@ -0,0 +1,83 @@
+import java.util.ArrayList;
+
+/**
+ * Bracket Class
+ * Created by Matt and Dan on 5/1/2017.
+ */
+public class Bracket {
+    //Attributes
+    ArrayList<String> bracket;
+    String name;
+    static final int EAST_BRACKET = 63;
+    static final int WEST_BRACKET = 79;
+    static final int NORTH_BRACKET = 95;
+    static final int SOUTH_BRACKET = 111;
+
+    //Constructor
+    /**
+     *Cosntructor using an ArrayList of strings to start
+     */
+    public Bracket(ArrayList<String> starting){
+        bracket = starting;

```

Comment:
im not really sure what the difference is doing it that way but I can fix that. Is it because bracket just becomes a pointer to starting rather than a unique entity?